src/Resources/views/Common/Label/refundPaymentState.html.twig

```diff
@@ -0,0 +1,13 @@
+{%
+    set viewOptions = {
+        completed: { icon: 'adjust', color: 'green' },
```

Comment:
Colors and icons are re-used from regular payment widget. Looks to me better balanced.